docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,38 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hollweg Dispersion Solver\n",
+    "\n",
+    "[tfds]: ../../api/plasmapy.dispersion.numerical.hollweg_.hollweg.rst\n",
+    "[bellan2012]: https://doi.org/10.1029/2012JA017856\n",
+    "[hollweg1999]: https://doi.org/10.1029/1998JA900132\n",
+    "\n",
+    "This notebook details the functionality of the [hollweg()][tfds] function.   This function computes the wave frequencies for given wavenumbers and plasma parameters.\n",
```

Comment:
```suggestion
    "This notebook details the functionality of the [hollweg()][tfds] function.   This function computes the wave frequencies for given wavenumbers and plasma parameters.\n"
```

I think this is the issue.  In notebooks (JSON files) the last line of an entry (list) can not have a comma.

Reply:
This fixed the issue and I can see the notebook now. I have a decent junk of it done explaining the equations and parameters and so forth. I'm going to address the check that isn't passing soon, I just wanted to get the ball rolling with writing the documentation.

Reply:
@rocco8773 I suppose we'll be discussing the specifics of what else to include in the notebook when we meet Friday, but I was going to work on code for reproducing Fig 2 in Hollweg1999 in the meantime. I think that's one thing we're definitely going to want to include right?

Reply:
@Sjbrownian I think that's a good idea.  That will show our reproducibility with existing published work.